roles/deploy_jhub/tasks/main.yml

```diff
@@ -1,14 +1,31 @@
-- name: Enable PV Claims on Cinder
-  kubernetes.core.k8s:
-    state: present
-    src: "{{ role_path }}/files/{{ storage_class_file }}"
+- name: Download the Cinder CSI Plugin
+  ansible.builtin.git:
+    repo: "https://github.com/kubernetes/cloud-provider-openstack.git"
+    dest: "/tmp/cloud-provider-openstack"
+    version: "release-1.24"
+    update: true # Automatically pull bug-fixes in
+
+- name: Install Cinder CSI
+  kubernetes.core.helm:
+    name: cinder-csi
+    namespace: kube-system
+    chart_ref: "/tmp/cloud-provider-openstack/charts/cinder-csi-plugin"
```

Comment:
We need to add the Openstack Manager steps as we cannot deploy without the LB support it provides:
https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

(I think the actual steps are in our Cluster API Docs)

Assume that the user has populated the secret, or we could double check it with Ansible (not sure how though)